src/main.cpp

```diff
@@ -214,6 +218,13 @@ static int populate_sota_dir(lmp_options &opt, ptree &resp, string &pkey)
 		out.close();
 	}
 
+	if (!opt.mprotect_key.empty()) {
+		/* Write the MProtect public key */
+		std::ofstream out(opt.sota_dir + "/mprotect_key.pem");
```

Comment:
Why do we need to store the pub key on a file system?

Reply:
_The key will be stored in /var/sota in PEM format (for reference)_ meaning, just for debugging purposes.

Reply:
ok.